app/editor/src/features/admin/charts/constants/defaultChartTemplate.ts

```diff
@@ -10,10 +10,7 @@ export const defaultChartTemplate: IChartTemplateModel = {
   isPublic: false,
   settings: {
     chartTypes: [],
```

Comment:
A chart template currently only has 3 configurable options.  We'll need to add more to cover labels, heading, colours, styling, etc.